offix/src/main/java/org/aerogear/offix/interceptor/InterceptorCallback.kt

```diff
@@ -0,0 +1,54 @@
+package org.aerogear.offix.interceptor
+
+import android.util.Log
+import com.apollographql.apollo.api.Error
+import com.apollographql.apollo.exception.ApolloException
+import com.apollographql.apollo.interceptor.ApolloInterceptor
+import org.aerogear.offix.ConflictResolutionHandler
+import org.aerogear.offix.responseWithConflicts
+import java.nio.charset.Charset
+
+class InterceptorCallback : ApolloInterceptor.CallBack {
+    private val TAG = javaClass.simpleName
+
+    override fun onResponse(response: ApolloInterceptor.InterceptorResponse) {
+        Log.d(TAG + 1, ConflictResolutionHandler().conflictPresent(response.parsedResponse).toString())
```

Comment:
Don't modify the tag, instead add the method name to the message to identify where the log is coming from.